services/app-api/utils/reports/index.ts

```diff
@@ -0,0 +1,76 @@
+import { getPossibleFieldsFromFormTemplate } from "../formTemplates/formTemplates";
+import s3Lib, { getFieldDataKey } from "../s3/s3-lib";
+import { AnyObject, State } from "../types";
+
```

Comment:
not sure where best to put this, but does this function account for if a field type changes but the key does not? For example: we change a text box to text area, or we change a radio to a checkbox, or radio to text area. We would probably not (?) want to copy in that case, right?

Reply:
per the spike: we probably wouldn't be altering a question - if a question needed updating, we would create a new question and any other validation changes to fields would fail on the front end.